src/components/shared/ARView/AnimatedCharacter/index.js

```diff
@@ -1,20 +1,17 @@
 // @flow
-
 import React from "react";
-
-import {
-  ViroNode,
-  ViroAmbientLight,
-  ViroDirectionalLight,
-  Viro3DObject,
-  ViroAnimations,
-  ViroMaterials,
-} from "react-viro";
+import { ViroNode, ViroAmbientLight, ViroDirectionalLight, Viro3DObject, ViroAnimations, ViroMaterials } from "react-viro";
 
 type Props = {
-  markerPosition: Array<number>
+  markerPosition: Array<number>,
 };
 
+const seagullModel = require("../../../../3D/seagull.vrx");
+const seagullTexture = require("../../../../3D/Mat_Seagull.png");
+const hatTexture = require("../../../../3D/Hat-512x512.png");
+const seagullNormalMap = require("../../../../3D/Hat-512x512-Normal.png");
```

Comment:
I wonder if we have to make these materials explicit in the ViroMaterials?

Reply:
Couldn't find any information on whether it was needed or not. But I added them and testet so I at least didn't break anything. Couldn't see any difference really but better safe than sorry